src/components/NftPreview.vue

```diff
@@ -0,0 +1,26 @@
+<script setup>
+const props = defineProps({
+  item: Object,
+  size: Number
+});
+
+function handleError(e) {
+  e.target.src = `https://cdn.stamp.fyi/token/${props.item.contract_address}?s=256`;
```

Comment:
I'd recommend to use a 2x size here to get more precision and optimize to not load too big image
```suggestion
  e.target.src = `https://cdn.stamp.fyi/token/${props.item.contract_address}?s=${props.size * 2}`;
```